cfme/common/provider_views.py

```diff
@@ -450,7 +450,11 @@ class endpoints(View):  # NOQA
 
     @property
     def is_displayed(self):
-        return self.logged_in_as_current_user
+        return all(
```

Comment:
**REQUIRED**
We shouldn't use `all()` in `is_displayed()`. When you use `all()` it checks each element in iterable. But whe you use something like that:
```python
return (
    self.title.is_displayed and
    self.name.is_displayed and
    ...
)
```
it will stop checking in the first false value.